kroxylicious/src/main/java/io/kroxylicious/proxy/internal/filter/FetchResponseTransformationFilter.java

```diff
@@ -85,8 +86,9 @@ public CompletionStage<ResponseFilterResult> onFetchResponse(short apiVersion, R
         if (!requestTopics.isEmpty()) {
             LOGGER.debug("Fetch response contains {} unknown topic ids, lookup via Metadata request: {}", requestTopics.size(), requestTopics);
             // Version 12 required for topic id support.
+            var metadataHeader = new RequestHeaderData().setRequestApiVersion((short) 12);
```

Comment:
Extract a constant for version 12? 

`TOPIC_ID_VERSION` ?

Reply:
Done.